services/system/Transact/src/RTransact.cpp

```diff
@@ -1421,6 +1421,7 @@ std::optional<AccountNumber> RTransact::getUser(HttpRequest request)
 {
    std::vector<char>            key = getJWTKey();
    std::optional<AccountNumber> result;
+   bool                         isDevChain = request.isDevChainOrigin();
```

Comment:
Can't RTransact just check for the presence of *auth-cookie* or __HOST-*auth cookie*? Then it wouldn't need to care about whether the origin is localhost or not.